src/pybind/mgr/volumes/fs/operations/clone_index.py

```diff
@@ -64,6 +64,16 @@ def get_oldest_clone_entry(self, exclude=[]):
                 sink_path = self.fs.readlink(os.path.join(self.path, min_ctime_entry[0]), CloneIndex.PATH_MAX)
                 return (min_ctime_entry[0], sink_path[:linklen])
             except cephfs.Error as e:
+                log.debug('Exception cephfs.Error has been caught. It appears that '
```

Comment:
The debug logs are same for this and `cephfs.ObjectNotFound` which does not make sense.

Reply:
Fixed.

Reply:
ping